src/components/MemberContribution/ContributionAccordianItem.tsx

```diff
@@ -7,15 +7,15 @@ import NextLink from "next/link";
 import { Button, Link } from "@chakra-ui/react";
 import styles from "./memberContribution.module.css";
 import Image from "next/image";
-import { useGetIsSuperUser } from "@/src/utils/customHooks";
+import { useGetIsSuperUser } from "../../utils/customHooks";
```

Comment:
why is this changed?

Reply:
As of now jest is not configured correctly in member site even for previous test i have change import